asteroid/losses/__init__.py

```diff
@@ -1,22 +1,26 @@
 from .pit_wrapper import PITLossWrapper
 from .mixit_wrapper import MixITLossWrapper
-from .sinkpit_wrapper import SinkPITLossWrapper, SinkPITBetaScheduler
+from .sinkpit_wrapper import SinkPITLossWrapper
 from .sdr import PairwiseNegSDR
 from .sdr import pairwise_neg_sisdr, singlesrc_neg_sisdr, multisrc_neg_sisdr
 from .sdr import pairwise_neg_sdsdr, singlesrc_neg_sdsdr, multisrc_neg_sdsdr
 from .sdr import pairwise_neg_snr, singlesrc_neg_snr, multisrc_neg_snr
 from .mse import pairwise_mse, singlesrc_mse, multisrc_mse
 from .cluster import deep_clustering_loss
 from .pmsqe import SingleSrcPMSQE
-from .stoi import NegSTOILoss as SingleSrcNegSTOI
 from .multi_scale_spectral import SingleSrcMultiScaleSpectral
 
+try:
+    from .stoi import NegSTOILoss as SingleSrcNegSTOI
+except ModuleNotFoundError:
+    # Is installed with asteroid, but remove the deps for TorchHub.
+    SingleSrcNegSTOI = lambda *a, **kw: print("Install `torch_stoi` to use `SingleSrcNegSTOI`")
+
```

Comment:
Point of doubt

Reply:
Better raise an error rather than printing only

Reply:
Yes, you're right, I'll do that.